iidm/iidm-mergingview/src/test/java/com/powsybl/iidm/mergingview/TopologyTest.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.iidm.mergingview;
+
+import com.powsybl.iidm.network.Line;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.VoltageLevel;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Mathieu Bague <mathieu.bague@rte-france.com>
+ */
+public class TopologyTest {
+
+    @Test
+    public void test() {
+        MergingView cgm = MergingViewFactory.createCGM(null);
+        Network n1 = cgm.getNetwork("n1");
+        Network n2 = cgm.getNetwork("n2");
+        assertNotNull(n1);
+        assertNotNull(n2);
+
+        Line mergedLine = cgm.getLine("DL1 + DL2");
+        assertNotNull(mergedLine);
+        assertNotNull(mergedLine.getTerminal1().getBusView().getBus());
+        // FIXME(mathbagu) assertNotNull(mergedLine.getTerminal2().getBusView().getBus());
```

Comment:
Hi @mathbagu ! I have tried to resolve this fixme and it is fix with this PR https://github.com/powsybl/powsybl-core/pull/1316/files

Reply:
Indeed, now the PR that resolves this test is #1340